scripts/setup_bridge.py

```diff
@@ -0,0 +1,24 @@
+import shutil
+
+from setuptools import setup
+from setuptools_rust import Binding, RustExtension
+
+# Same as in /build.py, but we want this self-contained
+rust_extensions = [
+    RustExtension(
+        "temporalio.bridge.temporal_sdk_bridge",
+        path="temporalio/bridge/Cargo.toml",
+        binding=Binding.PyO3,
+        py_limited_api=True,
+        features=["pyo3/abi3-py37"],
```

Comment:
The stuff I mentioned to make sure we get small sizes can be added to the bridge cargo like so

https://github.com/temporalio/sdk-typescript/blob/main/packages/core-bridge/Cargo.toml#L13

Assuming this builds with release by default, then it will automatically use those settings.

Reply:
Done